GVRf/Framework/framework/src/main/jni/objects/components/render_data.h

```diff
@@ -42,19 +42,6 @@ class TextureCapturer;
 class RenderPass;
 struct RenderState;
 
-template<typename T>
-std::string to_string(T value) {
```

Comment:
Not sure how it built for me first-hand but now it doesn't anymore because glslang/Include/Common.h cannot be found. Since std::to_string is not available through the standard <string> header on Android for some reason, we should rather use our own version of to_string instead of something else non-standard.

Reply:
Probably one more reason to move to clang and libc++. Other than GCC going to be removed from NDK v18.